packages/react/src/component-tests/IcSwitch/IcSwitchTestData.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState, useRef } from "react";
-import { IcButton, IcSwitch } from "../../components";
+import { IcButton, IcSwitch, IcTypography } from "../../components";
```

Comment:
```suggestion
import { IcButton, IcSwitch } from "../../components";
```
IcTypography doesn't appear to be used